nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/main/java/org/apache/nifi/processors/kudu/PutKudu.java

```diff
@@ -169,6 +145,13 @@
         .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
         .build();
 
+    protected static final PropertyDescriptor IGNORE_NULL = new Builder()
+        .name("Ignore NULL")
+        .description("Ignore NULL on Kudu Put Operation, Update only non-Null columns if set true")
+        .defaultValue("false")
+        .allowableValues("true", "false")
```

Comment:
Not an issue but more for consistency: usually when we have a property using ``.allowableValues`` we also add ``.required(true)``.